pypiserver/_app.py

```diff
@@ -362,6 +363,25 @@ def server_static(filename):
 
     return HTTPError(404, f"Not Found ({filename} does not exist)\n\n")
 
+@app.route("/:project/json")
+@auth("list")
+def json_info(project):
+    packages = sorted(
+        config.backend.find_project_packages(project),
```

Comment:
I noticed that at `/:project` (which redirects to `/simple/:project`) pypiserver accepts only normalized project names according to [PEP 503](https://www.python.org/dev/peps/pep-0503/). Might be a detail, but I think it'd be good if we could keep this behaviour consistent. :) 

For instance, you could add a check similar to how it is implemented at [`_app.py:L279`](https://github.com/pypiserver/pypiserver/blob/master/pypiserver/_app.py#L279).

```python
@app.route("/simple/:project/")
@auth("list")
def simple(project):
    # PEP 503: require normalized project
    normalized = normalize_pkgname_for_url(project)
    if project != normalized:
        return redirect(f"/simple/{normalized}/", 301)
    ...
```

Reply:
Perfect, thanks for addding!